spec.md

```diff
@@ -1617,6 +1644,53 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 |-----------|-----------|-------------|-------------------|
 | Volume does not exist | 5 NOT_FOUND | Indicates that a volume corresponding to the specified `volume_id` does not exist. | Caller MUST verify that the `volume_id` is correct and that the volume is accessible and has not been deleted before retrying with exponential back off. |
 
+
+#### `ControllerModifyVolume`
+
+A Controller plugin MUST implement this RPC call if the plugin has the MODIFY_VOLUME controller capability.
+This RPC allows the CO to change mutable key attributes of a volume. 
+
+This operation MUST be idempotent. 
+The new mutable parameters in ControllerModifyVolume can be different from the existing mutable parameters. 
```

Comment:
What does this mean?

Reply:
This means that, for example the existing mutable parameters in the volume has a field iops, and it is set to 500. The new mutable parameters in ControllerModifyVolume can set this field iops to something diff, like 1000.